compose-files/aquarius.yml

```diff
@@ -30,5 +30,6 @@ services:
       EVENTS_RPC: "http://172.15.0.3:8545"
       EVENTS_ALLOW: "0"
       RUN_EVENTS_MONITOR: "1"
+      AQUARIUS_URL: http://172.15.0.5:5000
```

Comment:
we use `AQUARIUS_URI` everywhere else

Reply:
it's correct. URL is a subset of URI, and in our case, it's the binding URL for swagger